apps/comps/package.json

```diff
@@ -97,6 +97,7 @@
     "typedoc-plugin-coverage": "^3.4.1",
     "typedoc-plugin-markdown": "^4.6.0",
     "typescript": "^5.7.3",
+    "vite": "6.3.5",
```

Comment:
Not sure why we need this.

Reply:
Was necessary due to otherwise conflicting deps when adding `alchemy-sdk`